### PR TITLE
Add 'confirmations' to response of eth/header api

### PR DESCRIPTION
**Pull request by wuminzhe**
this will be used by dj's guard.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -46,6 +46,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 structopt = "0.3.17"
 uint = "0.8.3"
+web3 = { git = "https://github.com/tomusdrw/rust-web3" }
```
Comment (clearloop) [inline Cargo.toml:49]: Crate can not publish to `crates.io` if using `git` resource.

Comment (wuminzhe) [reply Cargo.toml:49]: I'll change to direct http request

File: src/api/eth/header.rs [hunk 2]
```diff
@@ -32,12 +39,37 @@ pub async fn handle(block: web::Path<String>, shared: web::Data<ShadowShared>) -
...
+		.into(),
+		mmr_root: format!("0x{}", root),
+	};
+
+	let confirmations = get_confirmations(num).await.unwrap_or(0);
+
+	web::Json(Resp {
+		proof: proof,
+		confirmations: confirmations
+	})
```
Comment (clearloop) [inline src/api/eth/header.rs:56]: This resp will break the guard API of dj

File: src/api/eth/header.rs [hunk 3]
```diff
@@ -32,12 +39,37 @@ pub async fn handle(block: web::Path<String>, shared: web::Data<ShadowShared>) -
...
+	let current_height = web3.eth().block_number().await?;
 
-    web::Json(ProofResp {
-        header: EthHeader::get(&shared.client, num)
-            .await
-            .unwrap_or_default()
-            .into(),
-        mmr_root: format!("0x{}", root),
-    })
+	Ok(current_height.as_u64() - block_number)
```
Comment (clearloop) [inline src/api/eth/header.rs:74]: This part of logic should be placed in `/src/chain/eth/confirmation.rs`

File: src/api/eth/header.rs (hunk 2 above)
Comment (wuminzhe) [reply src/api/eth/header.rs:56]: yes, maybe put this to dj is a better choice.

**Review by clearloop — APPROVED** [review]

**Pull request merged**